packages/swarm-docs/src/pages/NumericalInput.mdx

```diff
@@ -2,11 +2,12 @@ import { NumericalInput } from '@meetup/swarm-components';
 import Example from '../components/numericalExample';
 
 # Numerical Input
+Fills to the width of its container.
 ```.jsx
-<NumericalInput value={1} />
+<div style={{ width: 300 }}>
+  <NumericalInput value={1} />
```

Comment:
Is `onIncrement` still valid? Keeping it in the examples would be good

Reply:
It's not. It was refactored out.